deploy/helm/values.yaml

```diff
@@ -6,6 +6,20 @@ spoImage:
   repository: k8s-staging-sp-operator/security-profiles-operator
   tag: latest
 
+selinuxdImage:
+  default:
+    registry: quay.io
+    repository: security-profiles-operator/selinuxd
+    tag: latest
```

Comment:
We can also add security-profiles-operator/selinuxd-fedora :)

Reply:
There's no environment variable which reads this however

Reply:
Perhaps this will do? fd182ac7b5c7b3d6907eab5280d5015d8d8657e7

Reply:
Yep, thanks!